src/fifo-queue/fifo-queue-dynamodb.ts

```diff
@@ -60,4 +60,15 @@ export class FifoQueueDynamodb implements FifoQueue {
       throw err;
     }
   }
-}
+
+  async isEmpty(): Promise<boolean> {
+    const params = {
+      TableName: this.tableName,
+      Select: "COUNT",
+    };
+    const res = await this.db.scan(params).promise();
```

Comment:
Use `this.db.query` instead. scan is a costly operation (it will scan the whole database) and don't use the indexes